tableaudocumentapi/connection.py

```diff
@@ -30,18 +30,20 @@ def __init__(self, connxml):
         self._username = connxml.get('username')
         self._authentication = connxml.get('authentication')
         self._class = connxml.get('class')
+        self.port = connxml.get('port', None)
```

Comment:
I think originally I didn't so that validation logic would get triggered, but with my new "Don't write it" approach I think I can do just `_port` now.